1.1.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+//�������� ���������, ��������� �������� ������� x4 + x3 + x2 + x + 1 �� ��� ���������.
+
+float f(float x)
+{
+	float y = x * x;

```

Comment:
```suggestion
	float xSquare = x * x;
```
У переменных должны быть говорящие имена.